app/models/document/delegation_service_public.rb

```diff
@@ -0,0 +1,2 @@
+class Document::DelegationServicePublic < Document
+end
```

Comment:
Is this intentional to have this empty file ?

Reply:
ruby « magic » !